mux/bindings/python/cmux_mux_client/client.py

```
@@ -0,0 +1,541 @@
+        self.response = response
+
+
+class MuxConnectionError(MuxError):
+    pass
+
+
+class ProtocolError(MuxError):
+    pass
+
+
+class TimeoutError(MuxError):
```

Ruff (0.15.20)

[error] Variable `TimeoutError` is shadowing a Python builtin

(A001)